public/config/ex-lign101/format.sample.yaml

```diff
@@ -18,3 +18,4 @@ words:
       - correct_value: true
         wrong_value: incorrect message
         ...: anything else message
+        HINT: an optional hint for this segment if the user gets it wrong too many times (e.g. just isn't hearing glottal stop)
```

Comment:
@stylerw does this look like what you had in mind for the HINT functionality? You can test it currently at https://add-hints.int-ipa.pages.dev/#/class/ex-lign101/assignment1. I made the syntax all caps on the key so it stands out a bit more (case-sensitive). Also, it is treated specially, so a student can't just type "HINT" to get the explanation. As a consequence of that, it doesn't work as part of a macro, but I think that's reasonable behaviour.